### DISCUSSION TITLE
Unity数据持久化-json - 没有秃头基因的燊的博客

### DISCUSSION THREAD

**Discussion by giscus** [yzs020220/giscus-repo#15] [Announcements]
# Unity数据持久化-json - 没有秃头基因的燊的博客

谈谈关于如何在Unity中将数据转为json文件来进行存储与读取，使用的都是JsonUtility中的方法，非常的便捷 - 没有秃头基因的燊 - 没有秃头基因的燊的博客

https://yzs020220.github.io/posts/30759/

**Comment by HCY-ASLEEP** [comment]
如果想要存取速度更快的数据格式，可以选择 toml 或者 yaml

**Reply by yzs020220** [reply]
确实有yaml的，我有空也可以看看，不过其实主要是数据恢复到实例这一步骤最难写，到头来都基本是自己写
